disnake/ext/commands/bot_base.py

```diff
@@ -141,6 +146,20 @@ def __init__(
     ):
         super().__init__(**options)
         self.command_prefix = command_prefix
+        if (
+            command_prefix is not when_mentioned
+            and not self.intents.message_content  # type: ignore
+            and BotBase._warn_prefix
+        ):
+            BotBase._warn_prefix = False
+            _log.warning(
```

Comment:
this won't show up if logging isn't set up

Reply:
You should use a Warning?

Reply:
ty for the suggestion, implemented in https://github.com/DisnakeDev/disnake/pull/353/commits/2a180ee5d932aa45d99ab7f0d5287cb49e88792d